### ISSUE TITLE
Add ability to place attachments in table cells

### ISSUE THREAD

**Issue by tommoor** [outline/outline#5981]
### Discussed in https://github.com/outline/outline/discussions/5980

<div type='discussions-op-text'>

<sup>Originally posted by **sorvis** October 10, 2023</sup>
It would nice to have the ability to place uploaded files into a table.

![image](https://github.com/outline/outline/assets/1037467/a5d76e84-6f2a-4bd9-bd37-bdf5509f7450)
</div>

**Comment by tommoor** [comment]
Dupe of #5586

**Comment by ultimate1112** [comment]
Hey @tommoor !
Mind if I try creating this feature?
I was thinking of opening up TableCell to allow all "block" elements, such as Task Lists, Attachments and more... Similar to how blockquotes work.

**Comment by ultimate1112** [comment]
You'll probably already know this, but to conform to the markdown style of Outline, we'll have to encapsulate this rich content in html.
https://blog.jakelee.co.uk/displaying-complex-content-inside-jekyll-tables/

**Comment by tommoor** [comment]
Yeah, it's basically blocked by the completion of #3000 - so that the source of truth for tables is not markdown. Much more content can be supported in the cell then.
